### ISSUE TITLE
Automatically enabling `--ozone-platform=*` breaks Webcord on Wayland

### ISSUE THREAD

**Issue by alxpettit** [NixOS/nixpkgs#263557]
This is in reference to the latest version of the Webcord package (master as of commit #0178524).

### Steps To Reproduce
1. Get a machine with a NVIDIA TU116M or similar
2. try to run Webcord with no profile (`mv ~/.config/Webcord ~/.config/Webcord.tmp`)
3. Attempt execution via calling `webcord`. It crashes perpetually with identical output to [this issue](https://github.com/SpacingBat3/WebCord/issues/433)

### Expected behavior
No crash

### Additional context

Note that this is a hardware-specific problem, and thus will likely not be possible to reproduce for the package maintainers, unless they happen to have some NVIDIA hardware that can trigger the bug. Electron's support for Wayland has been spotty for a long time, and hardware-specific issues like this are common. :/

I recommend removing or adjusting the behavior of automatically prepending any sort of `--ozone-platform` flag. While it is sensible and it _should_ work, I can report that upstream support is still too shoddy to rely on this. :(

Note that I _have_ run proper testing, including copying the contents of the wrapper script and editing the flag out. These modified wrapper scripts confirm that the `--ozone-platform` flag is the issue.

### Notify maintainers

@huantian @eclairevoyant

### Metadata
```
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.55, NIXOS, 23.05 (Stoat), 23.05.20231003.8a4c174`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.5`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by eclairevoyant** [comment]
What is `commit #178524`? That is not a commit sha, nor has anything to do with webcord.

Anyway, this has already been fixed in #262343, I suggest waiting until that lands.

**Comment by alxpettit** [comment]
Oops. Grabbed the pull number instead of commit number. Oh well.

Yes, that's useful. No need to wait for upstream merge. I'll just pull a master flake input. Thanks for the info :P
